src/templates/cashier/deposit_withdraw.html.tt

```diff
@@ -0,0 +1,59 @@
+<div class="fx-section">
+    <div id="cashier_deposit" class="container">
+        <div class="row">
+            <div class="col-sm">
+                <h1 id="cashier_title" style="text-transform: capitalize"></h1>
+            </div>
+        </div>
+
+        <div class="row">
+            <div id="error_msg" class="col-xs-12 notice-msg center-text hidden"></div>
+            <div id="ukgc_funds_protection" class="hidden">
```

Comment:
@kellybinary   ukgc_funds_protection is for MX accounts. you don't need to handle this in champion-fx. remind me to explain this tomorrow.